.github/workflows/python-app.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install jupyter-client==6.1.6
+        pip install wheel
+        pip install -r requirements.txt
+        pip install -r requirements-dev.txt
+    - name: Lint check with black
+      run: |
+        black --target-version py37 --line-length 105 --check .
+    - name: Test with pytest
+      run: |
+        python -m pytest tests/*.py
+    - name: Code Coverage Report
+      run: |
+        pytest --cov-report term --cov=lux tests/
```

Comment:
I know that this is probably copied over from what we originally had in the `.travis.yml`, but it looks like this line rerun the test suite again with the coverage report? 
Could we combine this line with the earlier `python -m pytest tests/*.py` so that we only need to run the test suite once?

Reply:
Let me check.